tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf

```diff
@@ -16,7 +16,6 @@ Object.Pipeline {
 	mixout-gain-dai-copier-playback.0 {
 		index 2
 		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.2.1'
```

Comment:
yes, if we're not setting the stream name, we can remove the Object completely

Reply:
done